consul/consul_types.bal

```diff
@@ -72,7 +72,7 @@ public type HealthCheck record {
     string definition = "";
     int createIndex = 0;
     int modifyIndex = 0;
-};
+|};
```

Comment:
Any particular reason to change these records to closed records? No issue since we are manually mapping these fields at the moment, but is there a specific reason?